[RFC] vim-patch:7.4.{1053,1071}

jamessan opened the pull request:
**vim-patch:7.4.1053**
Problem:    Insufficient testing for quickfix commands.
Solution:   Add a new style quickfix test. (Yegappan Lakshmanan)

https://github.com/vim/vim/commit/da59dd5da6440c3410866ed61ce169a2012ba5bd

**vim-patch:7.4.1071**
Problem:    New style tests are executed in arbitrary order.
Solution:   Sort the test function names. (Hirohito Higashi)
            Fix the quickfix test that depended on the order.

https://github.com/vim/vim/commit/cfc0a350a9fa04f1b0cfa1ba31fbd2847376513f

jamessan committed:
vim-patch:7.4.1053

Problem:    Insufficient testing for quickfix commands.
Solution:   Add a new style quickfix test. (Yegappan Lakshmanan)

https://github.com/vim/vim/commit/da59dd5da6440c3410866ed61ce169a2012ba5bd

KillTheMule commented on test/functional/legacy/quickfix_spec.lua:
```diff
@@ -2,16 +2,294 @@
 
 local helpers = require('test.functional.helpers')
 local source, clear = helpers.source, helpers.clear
+local execute, nvim, call = helpers.execute, helpers.meths, helpers.call
```
Execute isn't used in this file.

jamessan replied:
Yeah, that's left over from when I was fiddling around with some failures.

KillTheMule commented on test/functional/legacy/quickfix_spec.lua:
```diff
@@ -2,16 +2,294 @@
 local source, clear = helpers.source, helpers.clear
+local execute, nvim, call = helpers.execute, helpers.meths, helpers.call
+
+local function expected_empty()
```
This function isn't used. You can tell because you did not provide `eq` :)

jamessan replied:
It's supposed to be, but I removed the calls when I was trying to figure out a hang.

KillTheMule commented:
I've added line comments, but `make testlint` should provide usefull output by now.

jamessan commented:
@jszakmeister Is the OSX failure here a sporadic issue or something that needs to be looked at on that system?

jamessan committed:
vim-patch:7.4.1071

Problem:    New style tests are executed in arbitrary order.
Solution:   Sort the test function names. (Hirohito Higashi)
            Fix the quickfix test that depended on the order.

https://github.com/vim/vim/commit/cfc0a350a9fa04f1b0cfa1ba31fbd2847376513f
